receiver/cloudfoundryreceiver/converter.go

```diff
@@ -41,15 +47,34 @@ func convertEnvelopeToMetrics(envelope *loggregator_v2.Envelope, metricSlice pme
 	}
 }
 
+func convertEnvelopeToLogs(envelope *loggregator_v2.Envelope, logSlice plog.LogRecordSlice, startTime time.Time) error {
+	log := logSlice.AppendEmpty()
+	log.SetTimestamp(pcommon.Timestamp(envelope.GetTimestamp()))
+	log.SetObservedTimestamp(pcommon.NewTimestampFromTime(startTime))
+	logLine := string(envelope.GetLog().GetPayload())
+	log.Body().SetStr(logLine)
+	//exhaustive:enforce
+	switch envelope.GetLog().GetType() {
+	case loggregator_v2.Log_OUT:
+		log.SetSeverityText(plog.SeverityNumberInfo.String())
+		log.SetSeverityNumber(plog.SeverityNumberInfo)
+	case loggregator_v2.Log_ERR:
+		log.SetSeverityText(plog.SeverityNumberError.String())
+		log.SetSeverityNumber(plog.SeverityNumberError)
```

Comment:
Since `stderr` can be used for anything related to potential problems, should we maybe downgrade the severity here to `warning` instead of `error`? I'm worried about causing more alarm than necessary, but on the flip side, we don't want to hide real problems.

I couldn't find anything in cloud foundry docs after a quick search to know if only errors are sent to stderr.

Reply:
We receive either error or info logs from cloudfoundry and we just map them here. We didn't want to change this convention.